openshift/kustomize/tekton/base/tasks/buildah.yaml

```diff
@@ -64,6 +64,9 @@ spec:
       script: |
         #!/usr/bin/env bash
 
+        export TMPDIR="/data/tmp/buildah"
```

Comment:
The docker contain was running out of space when building the API.  This places the build on the mapped volume instead.